tests/crypto/asn1_tests.c

```diff
@@ -264,8 +264,6 @@ static oe_result_t _parse(oe_asn1_t* asn1)
                 if (result != OE_OK)
                     goto done;
 
-                result = _parse(&sequence);
```

Comment:
I think this line is supposed to be kept, it is trying to recursively parse sequence.

The missing thing here is that the response isn't used. So the whole case should be:
```
case OE_ASN1_TAG_SEQUENCE:
{
    oe_asn1_t sequence;

    result = oe_asn1_get_sequence(asn1, &sequence);
    if (result != OE_OK)
        goto done;

    result = _parse(&sequence);
    if (result != OE_OK)
        goto done;

    break;
}
```